third_party/2and3/requests/models.pyi

```diff
@@ -117,6 +117,7 @@ class Response:
     def __iter__(self) -> Iterator[bytes]: ...
     def __enter__(self) -> Response: ...
     def __exit__(self, *args: Any) -> None: ...
+    def next(self) -> PreparedRequest: ...
```

Comment:
It looks as if this can be `None`, too, if there is no next request.

Reply:
During type check time we can't. We only know that method returns either `None` or a `PreparedRequest` instance. The caller has to check for `None` at runtime.

Reply:
Ah! Alrighty, fixed. Thank you! :)